api/v1alpha1/nificluster_types.go

```diff
@@ -495,6 +495,18 @@ type LdapConfiguration struct {
 	// Filter for searching for users against the 'User Search Base'.
 	// (i.e. sAMAccountName={0}). The user specified name is inserted into '{0}'.
 	SearchFilter string `json:"searchFilter,omitempty"`
+	// How the connection to the LDAP server is authenticated.
```

Comment:
@juldrixx : Are we adding new API changes to only `v1` or to both `v1` and `v1alpha1`?

Reply:
Should I remove it from `v1alpha` ?

Reply:
[#02b](https://github.com/konpyutaika/nifikop/pull/266/commits/02bdf77bd5ca544a34b74fb543950c48b4befc4d)